### DISCUSSION TITLE
When do we expect to have V1.23 available?

### DISCUSSION THREAD

**Discussion by Tangerino** [micropython/micropython#14480] [General]
V1.23 fixed some panic events. When do we expect to have its final release?
Thanks in advance

**Comment by sosi-deadeye** [comment]
I guess, when this is done: https://github.com/micropython/micropython/milestone/6

**Reply by Tangerino** [reply]
It is not available in MP web site along the release notes...

**Comment by mattytrentini** [comment]
[v1.23.0](https://github.com/micropython/micropython/releases/tag/v1.23.0) was released today.
